modules/ROOT/pages/cluster-add-srv.adoc

```diff
@@ -3,21 +3,20 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To add a server to a cluster, all servers in a cluster must run the same version of Mule runtime and of the Runtime Manager Agent.
+To add a server to a cluster, all servers in a cluster must run the same version of Mule runtime and of the Runtime Manager agent.
 
 [NOTE]
-Remember that you must first register each of the servers in the cluster by downloading and installing the same version of the https://www.mulesoft.com/platform/mule[Mule] and then running the xref:managing-servers.adoc#add-a-server[amc_setup] script on each.
-A cluster can only be made up of servers that are not already in a server group or part of another cluster.
+You must first register each of the servers in the cluster by downloading and installing the same version of the https://www.mulesoft.com/platform/mule[Mule] and then running the xref:managing-servers.adoc#add-a-server[amc_setup] script on each.
 
-
-. In the *Runtime Manager Servers* tab, click the name of a cluster to open the details view.
+. In Runtime Manager, in the menu on the left, click *Servers*. 
+. In the server list, under *Type*, click the name of the cluster to which to add the server. +
+The management panel opens on the right side of the page.
 +
 image::server-details.png[]
 
-. Click *Add Servers*.
-+
-image::add-servers-button.png[AddServers]
+. Click *Add Server*. +
+You can add only servers that are not already in a server group or part of another cluster.
```

Comment:
A little awkward? (just me?) Are servers already in a group/part grayed out? What prevents "me" from selecting them. I'd write it "Add  the servers you wish to include in the group. Servers that belong to another group or cluster are grayed out (or whatever indicates you can't choose them).  It's longer but the two concepts are broken out into two separate sentences ...

Reply:
Oh, and then you don't need most of the sentence in line 20